src/client/src/consumer.rs

```diff
@@ -410,12 +416,13 @@ impl PartitionConsumer {
             .lookup_version(DefaultStreamFetchRequest::API_KEY)
             .unwrap_or((WASM_MODULE_API - 1) as i16);
 
-        if !config.wasm_module.is_empty() {
-            if stream_fetch_version >= WASM_MODULE_API as i16 {
-                stream_request.wasm_module = config.wasm_module;
-            } else {
+        if let Some(wasm) = config.smartstream_filter {
+            println!("USING SMARTSTREAM WITH WASM SIZE: {}", wasm.binary.len());
```

Comment:
I think you want `debug!`.

Reply:
Ah, I actually need to remove that, good catch